internal/observability/graph.go

```diff
@@ -36,58 +36,57 @@ var (
 	_defaultGraphSize = 128
 	// Latency buckets for histograms. At some point, we may want to make these
 	// configurable.
-	_ms      = time.Millisecond
-	_buckets = []time.Duration{
-		1 * _ms,
-		2 * _ms,
-		3 * _ms,
-		4 * _ms,
-		5 * _ms,
-		6 * _ms,
-		7 * _ms,
-		8 * _ms,
-		9 * _ms,
-		10 * _ms,
-		12 * _ms,
-		14 * _ms,
-		16 * _ms,
-		18 * _ms,
-		20 * _ms,
-		25 * _ms,
-		30 * _ms,
-		35 * _ms,
-		40 * _ms,
-		45 * _ms,
-		50 * _ms,
-		60 * _ms,
-		70 * _ms,
-		80 * _ms,
-		90 * _ms,
-		100 * _ms,
-		120 * _ms,
-		140 * _ms,
-		160 * _ms,
-		180 * _ms,
-		200 * _ms,
-		250 * _ms,
-		300 * _ms,
-		350 * _ms,
-		400 * _ms,
-		450 * _ms,
-		500 * _ms,
-		600 * _ms,
-		700 * _ms,
-		800 * _ms,
-		900 * _ms,
-		1000 * _ms,
-		1500 * _ms,
-		2000 * _ms,
-		2500 * _ms,
-		3000 * _ms,
-		4000 * _ms,
-		5000 * _ms,
-		7500 * _ms,
-		10000 * _ms,
+	_bucketsMs = []int64{
+		1,
+		2,
+		3,
+		4,
+		5,
+		6,
+		7,
+		8,
+		9,
+		10,
+		12,
+		14,
+		16,
+		18,
+		20,
+		25,
+		30,
+		35,
+		40,
+		45,
+		50,
+		60,
+		70,
+		80,
+		90,
+		100,
+		120,
+		140,
+		160,
+		180,
+		200,
+		250,
+		300,
+		350,
+		400,
+		450,
+		500,
+		600,
+		700,
+		800,
+		900,
+		1000,
+		1500,
+		2000,
+		2500,
+		3000,
+		4000,
+		5000,
+		7500,
+		10000,
```

Comment:
`net/metrics` bundles this hand-tuned set of buckets, so we can replace this literal with a call to [`bucket.NewRPCLatency`](https://godoc.org/go.uber.org/net/metrics/bucket#NewRPCLatency).